Documentation/Usage.md

```diff
@@ -253,45 +297,6 @@ At this time you will need to make another module map package to represent syste
 
 For example, `libarchive` optionally depends on `xz`, which means it can be compiled with `xz` support, but it is not required. To provide a package that uses libarchive with xz you must make a `CArchive+CXz` package that depends on `CXz` and provides `CArchive`.
 
----
-
-## Build an Executable
-
-*Content to come.*
-
----
-
-## Create a Package
-
-Simply put: a package is a git repository with semantically versioned tags, that contains Swift sources and a `Package.swift` manifest file at its root.
-
-### Turning a Library Module into an External Package
```

Comment:
This feels weird and complicated to me. Even creating multiple modules is kind of a challenge for first time users, so I think we should just get rid of this.